packages/react/src/hooks/buckets.ts

```diff
@@ -287,38 +303,28 @@ export function useAddFile() {
 }
 
 export function useDeleteObject() {
-  const chainId = useChainId();
-  const contractAddress =
-    bucketManagerAddress[chainId as keyof typeof bucketManagerAddress];
-
   const { writeContract, writeContractAsync, ...rest } = useWriteContract();
 
-  const baseConfig = useMemo(
```

Comment:
this is the only thing i wasnt sure about. i remved the `useMemo` because we no longer have access to the `contractAddress` dependency, and instead, use the `bucket` directly.